### ISSUE TITLE
Implement user account lockout

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#93]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To be locked out from signing in after a certain number of failed sign-in attempts.

### Why

To mitigate the risk of brute-force attacks.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
